server/chat.ts

```diff
@@ -989,7 +989,7 @@ export class CommandContext extends MessageContext {
 				}
 				if (targetUser.settings.blockPMs &&
 					(targetUser.settings.blockPMs === true || !user.authAtLeast(targetUser.settings.blockPMs)) &&
-					!user.can('lock')) {
+					(!user.can('lock' || user.group === '*'))) {
```

Comment:
```suggestion
					(!user.can('lock') || user.group === '*') {
```

Reply:
Personally, I'd implement this by adding a `bypassBlockPMs` permission rather than hardcoding `*` rank.

Reply:
(Also, you should test your changes locally before opening a pull request -- that way, you could have caught the error I fixed in my suggestion.)